index.bs

```diff
@@ -933,16 +951,12 @@ To <dfn>check if an [=attribution source=] exceeds the unexpired destination lim
      * |record|'s [=attribution rate-limit record/source site=] and |source|'s [=attribution source/source site=] are equal
      * |record|'s [=attribution rate-limit record/reporting endpoint=] and |source|'s [=attribution source/reporting endpoint=] are equal
      * |record|'s [=attribution rate-limit record/expiry time=] is greater than |source|'s [=attribution source/source time=]
+1. Let |unexpiredDestinations| be an [=set/empty=] [=set=].
 1. For each [=attribution source=] |unexpiredSource| of |unexpiredSources|:
-    1. If |source|'s [=attribution source/attribution destination=] and |unexpiredSource|'s
-        [=attribution source/attribution destination=] are equal, then return false.
-    1. If |distinctDestinations| [=set/contains=] |unexpiredSource|'s
-        [=attribution source/attribution destination=], [=iteration/continue=].
-    1. [=set/Append=] |unexpiredSource|'s [=attribution source/attribution destination=] to
-        |distinctDestinations|.
-1. If |distinctDestinations|'s [=set/size=] is greater than or equal to
-    the user agent's [=max destinations covered by unexpired sources=], then return true.
-1. Return false.
+    1. Let |unexpiredDestinations| be the result of taking the [=set/union=] of |unexpiredDestinations| and |unexpiredSource|'s [=attribution source/attribution destinations=].
```

Comment:
unexpiredSource is a rate-limit record, rather than a source. Given we are adding a rate limit record for each destination, I think we can use set/append here rather than union.

Reply:
Done.